pkg/scheduler/api/job_info.go

```diff
@@ -321,7 +320,9 @@ func (ji *JobInfo) Clone() *JobInfo {
 	}
 
 	for _, task := range ji.Tasks {
-		info.AddTaskInfo(task.Clone())
+		newTask := task.Clone()
+		newTask.IsBackfill = CheckBackfill(newTask.Pod)
```

Comment:
Why do we need to set the value of `IsBackfill` explicitly? Shouldn't `task.Clone()` take care of it?

Reply:
variable "task" in task.Clone() comes from the scheduler cache. isbackfill is never set on the task variable in the cache, only in the snapshots.

Reply:
updated.